R/convert.R

```diff
@@ -30,6 +32,8 @@ pl_from_arrow = function(
     rechunk = TRUE,
     schema = NULL,
     schema_overrides = NULL) {
+  warning("`pl$from_arrow()` is deprecated and will be removed in 0.14.0. Use `as_polars_df()` or `as_polars_series` insead.")
```

Comment:
```suggestion
  warning("`pl$from_arrow()` is deprecated and will be removed in 0.14.0. Use `as_polars_df()` or `as_polars_series()` insead.")
```

Reply:
Oops, looks like you guys were quick to find it (but I was committed faster than I realized it)